api/osb/check_instance_visibility_plugin.go

```diff
@@ -114,35 +106,40 @@ func (p *checkVisibilityPlugin) checkVisibility(req *web.Request, next web.Handl
 				StatusCode:  http.StatusBadRequest,
 			}
 		}
-		for _, v := range visibilities.Visibilities {
-			if v.PlatformID == "" { // public visibility
-				return next.Handle(req)
-			}
-			if v.PlatformID == platform.ID {
-				if v.Labels == nil {
-					return next.Handle(req)
-				}
-				orgGUIDs, ok := v.Labels["organization_guid"]
-				if !ok {
-					return next.Handle(req)
-				}
-				if slice.StringsAnyEquals(orgGUIDs, payloadOrgGUID) {
-					return next.Handle(req)
-				}
-			}
+
```

Comment:
why do you need named query here query.ByField + query.ByLabel won't do the job?